Example/Tests/JSONApiTesting.swift

```diff
@@ -61,12 +47,20 @@ class JSONApiTesting: XCTestCase {
 
     final class Author: PSJSONApiModel {
 
+        open var name: String = ""
+        open var age: Int = 0
+        open var gender: String = ""
+        open var articles: [Articles] = []
         override class var modelName: String {
             return "authors"
         }
 
         override func register(attributes: inout [PSJSONAPIProperty], andRelationships relationships: inout [PSJSONAPIProperty]) {
-
+            attributes.append(PSAttribute(property: &self.name, jsonKey: "name"))
```

Comment:
I've been meaning to ask about the need for `&` here. Is it passing the memory address? why?